### PR TITLE
Bahasa Indonesia Translation

### PR DESCRIPTION
**Pull request by rizasaputra**
Translation to Bahasa Indonesia for lab instructions

### CODE REVIEW

File: 1-lab-basicDispatchConsumeEvent/README.md [hunk 1]
```diff
@@ -128,15 +130,15 @@ cdk deploy
 #### Invoking Producer Function
 Now we are going to test the Producer function which will trigger an event to Amazon EventBridge and will be consumed by the Consumer function as well as print the logs into CloudWatch. 
 
-- Create a test event by choosing **Test** in the upper right corner
+- Create a test event by clicking **Test** button in the Lambda code editor menubar
```
Comment (rizasaputra) [inline 1-lab-basicDispatchConsumeEvent/README.md]: Updated layout in Lambda console

File: 1-lab-basicDispatchConsumeEvent/README.md [hunk 2]
```diff
@@ -128,15 +130,15 @@ cdk deploy
...
+- Create a test event by clicking **Test** button in the Lambda code editor menubar 
 - In the Configure test event page, choose **Create new test event** and in Event template, leave the default Hello World option. Enter an Event name and provide an empty template:
 ```json
 {}
 ``` 
 - Choose **Create** and then choose **Test**.
 
 #### Let's check the log on Amazon CloudWatch Logs
-- Navigate to Amazon CloudWatch [dashboard](https://ap-southeast-1.console.aws.amazon.com/cloudwatch/home).
+- Navigate to Amazon CloudWatch Log groups [dashboard](https://ap-southeast-1.console.aws.amazon.com/cloudwatch/home?region=ap-southeast-1#logsV2:log-groups).
```
Comment (rizasaputra) [inline 1-lab-basicDispatchConsumeEvent/README.md:142]: added `Log groups` to clarify

File: 1-lab-basicDispatchConsumeEvent/README-id.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
+# Lab 1: Pengiriman dan Penerimaan Dasar
+[English](README.md) | [Bahasa Indonesia](README-id.md)
+
+Dalam _workshop_ ini, Anda akan membangun 2 fungsi AWS Lambda. Satu fungsi AWS Lambda (sebagai Produsen) akan mengirimkan (_dispatch_) _event_, dan satu Fungsi AWS Lambda (sebagai Konsumen) akan menerima (_consume_) _event_. Selain menerima _event_, Konsumen juga akan menulis log ke AWS CloudWatch Logs untuk memastikan aliran prosesnya berjalan dengan baik.
+
+## Diagram
+![Diagram Lab 1](lab1-diagram-id.png)
```
Comment (rizasaputra) [inline 1-lab-basicDispatchConsumeEvent/README-id.md:7]: I use translated diagram (for lab 1 only)

**Review by rizasaputra — COMMENTED** [review]
Highlights on changes on original English instructions for easier review

File: 1-lab-basicDispatchConsumeEvent/README.md (hunk 1 above)
Comment (donnieprakoso) [reply 1-lab-basicDispatchConsumeEvent/README.md]: We need to adjust this particular instruction to adapt to recent UI changes. 
Here's what I have in mind:

- Navigate to the **Test** page by clicking the **Test** tab.
- In the Configure test event page, choose "New Event" and in Event template, leave the default Hello World option.
- Enter an Event name and provide an empty template:

File: 1-lab-basicDispatchConsumeEvent/README-id.md (hunk 3 above)
Comment (donnieprakoso) [reply 1-lab-basicDispatchConsumeEvent/README-id.md:7]: Let's stick with "Producer" (as opposed to Produsen) and "Consumer" (as opposed to Konsumen) and italicize those 2 words.

File: 1-lab-basicDispatchConsumeEvent/README.md (hunk 1 above)
Comment (rizasaputra) [reply 1-lab-basicDispatchConsumeEvent/README.md]: updated

File: 1-lab-basicDispatchConsumeEvent/README-id.md (hunk 3 above)
Comment (rizasaputra) [reply 1-lab-basicDispatchConsumeEvent/README-id.md:7]: updated

**Comment by donnieprakoso** [comment]
Thank you for you contribution @rizasaputra. All good!

**Pull request merged**